docs/variables.md

```diff
@@ -313,3 +314,17 @@ myMultiNestedMapping: {
   ...
 },
 ```
+
+## User defined types
```

Comment:
nit: User Defined Types -> [User-Defined Value Types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (including the embedded link to the Solidity docs)